### DISCUSSION TITLE
Support Blocker

### DISCUSSION THREAD

**Discussion by MakerPay** [supermerill/SuperSlicer#1975] [Q&A]
To start I would like to say I love SuperSlicer and it has worked great for my needs till now.  Not sure if this is getting posted in the correct place so if not please direct me to the proper place.  I am working on a product and I think something is broke in Super Slicer.....or I just cannot get it to work.  I am trying to put two objects in the same area with one hanging over the other.  I am using the lower object in lou of using support.  My hope was to get the support to build on top of the lower object for the upper but it runs the support all the way through the lower object to the build plate.  I tried to use the support blocker slab but it has no effect.  I can however use the support blocker cube and it will block support but as soon as I change the shape of that cube its ability to block the support goes away.  What I want to do works in idea maker but I really do not want to learn how to use that program.  Help?  Thanks in advance
![2-model-combined](https://user-images.githubusercontent.com/92654286/143797405-e1d44e14-9ea6-460a-b62f-85b08f8f0c24.JPG)
![2-model-infill-blocker-no-worky](https://user-images.githubusercontent.com/92654286/143797417-7415ba76-8690-40ab-a1de-34b1bc537e50.JPG)

.

**Comment by neophyl** [comment]
I know this was answered in an issue, but that's now closed so in the interest of anyone coming along and finding this question later here was my reply.

What you are posting about is normal operation for Super Slicer and Prusa Slicer.
As each Object is sliced individually then when you overlap them you get 2x extrusions in the same space, bad things happen. There are no checks for this, its the users responsibility.
The same is true for each Objects support too as its sliced with each object.

What you as a user need to do is to merge the Objects. Once its a single Object with multiple Parts then it will slice correctly and also stop putting support through the lower model. You can select both objects and right click>Merge.
